server/src/test/java/keywhiz/service/resources/automation/v2/BackfillRowHmacResourceClientIntegrationTest.java

```diff
@@ -0,0 +1,108 @@
+package keywhiz.service.resources.automation.v2;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jackson.Jackson;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.UUID;
+import keywhiz.IntegrationTestRule;
+import keywhiz.KeywhizService;
+import keywhiz.TestClients;
+import keywhiz.api.automation.v2.CreateSecretRequestV2;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static keywhiz.TestClients.clientRequest;
+import static keywhiz.client.KeywhizClient.JSON;
+import static org.junit.Assert.assertEquals;
+
+public class BackfillRowHmacResourceClientIntegrationTest {
+  private static final ObjectMapper mapper =
+      KeywhizService.customizeObjectMapper(Jackson.newObjectMapper());
+  private static final Base64.Encoder encoder = Base64.getEncoder();
+
+  OkHttpClient mutualSslClient;
+
+  @ClassRule public static final RuleChain chain = IntegrationTestRule.rule();
```

Comment:
We talked about naming in the previous commit, but I've taken it even further and added "Integration" to the name. Given that the test using the IntegrationTestRule, I feel like this is justified (and more clear than omitting it).